benchmarks/urls.py

```diff
@@ -43,6 +43,7 @@
     path('profile/submit/', user.Upload.as_view(domain="vision"), name=f'vision-submit'),
     path('profile/resubmit/', partial(user.resubmit, domain="vision"), name='vision-resubmit'),
     path('profile/logout/', user.Logout.as_view(domain="vision"), name='vision-logout'),
+    path('profile/nwb/', user.ConvertNWB.as_view(domain="vision"), name=f'convert-nwb'),
```

Comment:
@mike-ferguson to double check domain-specific url approach

Reply:
@mike-ferguson just checking if the endpoint is good as is or if it should be profile/vision/nwb/?